### PR TITLE
Change Addons to a list

### PR DESCRIPTION
**Pull request by Kn99HN**
From a suggestion in #29

### CODE REVIEW

File: kubernetes_env/kube_env.py
Comment (fruffy) [inline kubernetes_env/kube_env.py]: How does the input work? This is ideally a list right? I think this needs to change a little. Prometheus is also an addon.

**Review by fruffy — CHANGES_REQUESTED** [review]

File: kubernetes_env/kube_env.py
Comment (Kn99HN) [reply kubernetes_env/kube_env.py]: I now change it to letting user inputting addons name instead.

**Review by fruffy — APPROVED** [review]
Let's wait until the other request is merged before pulling this in.

**Pull request merged**
